assembl/static2/js/app/client.js

```diff
@@ -0,0 +1,13 @@
+import ApolloClient, { createNetworkInterface } from 'apollo-client';
+import { getDiscussionSlug } from './utils/globalFunctions';
+
+const client = new ApolloClient({
+  networkInterface: createNetworkInterface({
+    uri: `${window.location.origin}/${getDiscussionSlug()}/graphql`,
+    opts: {
+      credentials: 'same-origin'
```

Comment:
This is to send the authentication cookie in xhr requests, otherwhise you are anonymous in graphql view.